srcpkgs/wezterm/template

```diff
@@ -0,0 +1,68 @@
+# Template file for 'wezterm'
+pkgname=wezterm
+version=20220905
+revision=1
+_srcver=${version}-102802-7d4b8249
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="fontconfig-devel freetype-devel harfbuzz-devel openssl-devel
+ libX11-devel libxkbcommon-devel libxkbcommon-x11 wayland-devel xcb-util-devel
+ xcb-util-image-devel xcb-util-keysyms-devel xcb-util-wm-devel zlib-devel"
+depends="noto-fonts-emoji fonts-roboto-ttf"
+short_desc="GPU-accelerated cross-platform terminal emulator and multiplexer"
+maintainer="Joseph Benden <joe@benden.us>"
+license="MIT"
+homepage="https://wezfurlong.org/wezterm/"
+changelog="https://wezfurlong.org/wezterm/changelog.html"
+distfiles="https://github.com/wez/wezterm/releases/download/${_srcver}/wezterm-${_srcver}-src.tar.gz"
+checksum="5898af2bb2dbedcae2648764d5b7abd3d98b0aa3d05d171b09e0e3f76b7dd545"
+nocross=yes # ftbfs in libgit+libssh
```

Comment:
```suggestion
```
libgit2 is only used as a build dependency of wezterm-config, in the build.rs file (which always runs on host), so we have 2 options:

1. simple: add `libgit2-devel` to `hostmakedepends` so the build can work as-is (downside: it creates an unnecessary dependency for our build process)
2. a little more complex: patch out the relevant parts ([here](https://github.com/wez/wezterm/blob/20220905-102802-7d4b8249/config/build.rs#L17-L53) and [here](https://github.com/wez/wezterm/blob/20220905-102802-7d4b8249/config/Cargo.toml#L13-L14))

I think option 2 is the better one.

Reply:
No, notes buried inside Rust code discuss how mix-match of deps+system-deps causes no end of problems. In fact, I experienced them early on with wezterm packaging.

Reply:
Ok, the second option is best!